lib/package.json

```diff
@@ -32,9 +32,10 @@
     "test": "jest"
   },
   "dependencies": {
-    "@ledgerhq/wallet-api-client": "1.0.2",
+    "@ledgerhq/wallet-api-client": "1.1.0",
+    "@ledgerhq/wallet-api-core": "1.2.0",
```

Comment:
Don't we retrieve the "wallet-api-core" dependency via the transient dependency coming form "wallet-api-client"? This would avoid us mismatch with version management.

Reply:
You are totally right! done